CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 1.5.0
+
+### Updates (R4)
+
+* Added a `ResourceList` type for codes which are part of the [ResourceTypes](https://hl7.org/fhir/codesystem-resource-types.html) codesystem
+* Updated `Required` status for: MessageDefinition.focus.code, CapabilityStatement.rest.resource.type, GraphDefinition.start, GraphDefinition.link.target.type, ImplementationGuide.global.type, SearchParameter.base, SearchParameter.target, OperationDefinition.resource, CompartmentDefinition.resource.code, ExampleScenario.instance.resourceType
+* Changed `BundleEntry.resource` and other uses of generic `Resource` to a union type which supports inference and narrowing.
+
+## 1.4.10
```

Comment:
Is this version published? If the changes are rolled up into 1.5, then it's better to remove this and place it in the 1.5 section

Reply:
@TahaAttari

Reply:
updated 👍🏾